### ISSUE TITLE
Misleading "exiting" language in log output

### ISSUE THREAD

**Issue by mjustin** [mongock/mongock#231]
Very minor issue, but when I start up a Spring Boot application which uses Changock, and have Changock disabled, I find the log output to be a little misleading.  The last line of the application startup logs is the Changock log, which includes the wording "Exiting."  I realize that it's exiting the Changock process, but in context it looks more like it's exiting the Spring Boot application.

```
15:46:52.255 [main] INFO  o.s.b.a.e.web.EndpointLinksResolver - Exposing 13 endpoint(s) beneath base path '/actuator'
15:47:03.313 [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer - Tomcat started on port(s): 8080 (http) with context path ''
15:47:03.654 [main] INFO  o.s.d.r.c.DeferredRepositoryInitializationListener - Triggering deferred initialization of Spring Data repositories…
15:47:04.022 [main] INFO  o.s.d.r.c.DeferredRepositoryInitializationListener - Spring Data repositories initialized!
15:47:04.040 [main] INFO  c.e.c.c.example.Application - Started Application in 16.167 seconds (JVM running for 16.591)
15:47:04.044 [main] INFO  io.changock.runner.core.ChangockBase - Changock is disabled. Exiting.
```

Would it be better if the log statement used some other terminology, such as "Skipping", e.g. "Changock is disabled. Skipping migrations."?

The same logic probably applies to the other two places that use "Exiting" within [`ChangockBase`](https://github.com/cloudyrock/changock/blob/423ea1911d67c086384979253a8212fe4926a137/changock-runner/changock-runner-core/src/main/java/io/changock/runner/core/ChangockBase.java#L48):

```

      } catch (LockCheckException lockEx) {
        if (throwExceptionIfCannotObtainLock) {
          logger.error("Changock did not acquire process lock. EXITING WITHOUT RUNNING DATA MIGRATION", lockEx);
          throw new ChangockException(lockEx);

        } else {
          logger.warn("Changock did not acquire process lock. EXITING WITHOUT RUNNING DATA MIGRATION", lockEx);
        }

      }
```

Meta-question: For issues such as this that appear to be related to the `changock` libraries, do you want them reported against the [`cloudyrock/mongock`](https://github.com/cloudyrock/mongock) project or the [`cloudyrock/changock`](https://github.com/cloudyrock/changock) project?  The lack of a readme in `changock` suggests to me that perhaps it's intended to be only used internally, and that external issues should be filed against the `mongock` project, but that's just a guess based on limited context.

**Comment by dieppa** [comment]
Hey, we'll be reviewing all the logs and will take this into account. Thanks

**Comment by mjustin** [comment]
@dieppa 

Since you're reviewing the logs anyway, it seems a good time for me to share my other thoughts with respect to the logs I've been seeing.  Not sure if you want individual issues or one issue to group the various logging issues.  I'll add them in this comment for now, but I can split them out into individual issues if you'd like.

1. For the legacy migration, when the optional `changes-count-expectation` property is unset, a warn-level log is emitted: "[legacy-migration] - There is no changes count expectation!".  Given that this is an optional property, and one that is reasonable to leave unset, I would expect this to be at info or debug level (I think I personally would prefer "debug" as that makes it easier to disable the log).  Additionally, given the validity of disabling the property, it probably doesn't warrant an exclamation point in the message itself, as it's not necessarily indicative of something dramatic.
   1. In my particular case, the number of changes actually varies between my development environment and my production environment, as some of the changes we were testing out didn't end up making the cut to go to production, and were backed out before it reached that environment.  While we could modify the log history, not specifying the count seems the better approach.
2. In `MigrationExecutor`, all three change statuses (`APPLIED`, `PASSED OVER`, and `FAILED OVER`) are logged at info level.  Personally, I would like to be able to see the "APPLIED" & "FAILED OVER" ones without also seeing all the "PASSED OVER" ones, since the "APPLIED" & "FAILED OVER" ones show new changes (or issues), whereas the "PASSED OVER" values are an ever-growing list that doesn't change between runs.  However, this can't currently be accomplished, as all three are logged to info.  Would it make sense to move "PASSED OVER" to debug (and maybe "FAILED OVER" to warn)?
3. In `MigrationExecutor`, would "FAILED" be a better log output than "FAILED OVER"?

**Comment by dieppa** [comment]
Hello, thanks for your suggestions.

1. the changes-count-expectation is highly recommended. I am more inclined to be required by default and in case you want to ignore it, you must indicate so explicitly. The reason behind is that it would hurt much more if a you re-apply an undesired  change, than see the a log. If you are running a legacy migration, specially in production, you should be certain on how many changes you expect to be migrated. If not, you can exceptionally turn it off.
So in summary, I think the best approach is leaving the log like it is and probably make it required as default.

2. It makes sense to move the "PASSED OVER" to debug(or even trace) so you can turn it off while keeping the other two levels.

3. Yeah, could be.

Thanks
